src/test/java/io/jenkins/plugins/forensics/git/reference/GitCommitITest.java

```diff
@@ -55,11 +54,11 @@ public void shouldFindReferencePoint() throws Exception {
         JENKINS_PER_SUITE.assertBuildStatus(Result.SUCCESS, job.scheduleBuild2(0, new Action[0]));
 
         GitCommit jobGitCommit = job.getLastCompletedBuild().getAction(GitCommit.class);
-        assertEquals(3, jobGitCommit.getRevisions().size());
+        assertThat(3 == jobGitCommit.getRevisions().size());
```

Comment:
```suggestion
        assertThat(jobGitCommit.getRevisions()).hasSize(3);
```
Sonst wird gar nichts geprüft!